src/schema/generic.json

```diff
@@ -806,6 +806,13 @@
                 }
             ]
         },
+        "StructuredTextItems": {
```

Comment:
Can this be replaced with a `StructuredText` which already has a `content: Array<StructuredTextItem>`?

Reply:
How would you add a `NamedText` in here?

```
[{
    "type": "StructuredTextItem",
    "name": "TheName",
    "contents": [
        {
            "type": "NamedText",
            "name": "TheName",
            "text": "The text..."
        }
    ]
}]
```

It would require redundant nesting :/

Reply:
> It would require redundant nesting :/

with an implicit single item in the `contents` array